### DISCUSSION TITLE
Table 4

### DISCUSSION THREAD

**Discussion by nadiaBTS** [IBAHCM/RPiR#528] [Registration]
These are the links for table 4 group members' repositories.
Nadia Sheikh's: https://github.com/SBOHVM/nadiaBTSSeries01.git
@mhuzzell 
@Xinyu-GUO-2738378g 
@AnnaKinghorn 
@LollianKosyando

**Comment by mhuzzell** [comment]
Just commenting to confirm I can see the discussion!

Edit: and here's my repository: https://github.com/SBOHVM/mhuzzellSeries01.git

**Comment by Xinyu-GUO-2738378g** [comment]
My repository: https://github.com/Xinyu-GUO-2738378g/SeriesOne.git

**Comment by LollianKosyando** [comment]
Just commenting to confirm I can see the discussion!

**Comment by AnnaKinghorn** [comment]
Use this one not the other one https://github.com/SBOHVM/AnnaKinghorn1.git

**Comment by LollianKosyando** [comment]
Here is my repository
https://github.com/SBOHVM/lolliankosyandoSeries01.git

**Reply by nadiaBTS** [reply]
the link didnt work for me Lollian :(

**Reply by AnnaKinghorn** [reply]
Didn't work for me either

**Reply by mhuzzell** [reply]
Me neither. Did you share it to the group? I had trouble doing that at first -- you have to search 'rpir22', because for some reason just trying to put in SBOHVM/rpir22 doesn't work.

**Reply by LollianKosyando** [reply]
So sorry you guys couldn't have access to my repository. I have already fixed it, I think you can now access it.

**Reply by mhuzzell** [reply]
Yep, I can access it now -- thanks!

**Comment by mhuzzell** [comment]
Were we meant to close the issues after acknowledging them? I don't recall seeing an instruction one way or the other, but might have missed it... I'm leaving them open for now on mine.

**Reply by AnnaKinghorn** [reply]
Idk, I've left mine open too

**Reply by G-Shiach** [reply]
I think he said to close them on Monday

**Comment by G-Shiach** [comment]
Here is my new repository https://github.com/SBOHVM/G-ShiachSeries02.git

**Comment by AnnaKinghorn** [comment]
Here is my repository for series 2!
https://github.com/SBOHVM/AnnaKinghornPracticalSeries2.git
:)

**Reply by G-Shiach** [reply]
I can't open :(

**Reply by AnnaKinghorn** [reply]
I've added you now!

**Comment by nadiaBTS** [comment]
My repository for practical series 2:
https://github.com/SBOHVM/nadiaBTSSeries02

**Reply by AnnaKinghorn** [reply]
Hey, the link isn't working for some reason! Have you changed it in the privacy settings?

**Reply by nadiaBTS** [reply]
no I hadnt, thanks for pointing that out! I just changed it, hopefully it works now!

**Reply by AnnaKinghorn** [reply]
Yess that's it now!

**Comment by Xinyu-GUO-2738378g** [comment]
Morning! Here is my repository for practical series 2
 https://github.com/SBOHVM/XinyuSeries02.git

**Reply by AnnaKinghorn** [reply]
Hey, I can't open your's either, I think you need to change the privacy in settings to allow us to see it :)

**Reply by Xinyu-GUO-2738378g** [reply]
I added you to my repository! Please try again! : )

**Comment by G-Shiach** [comment]
https://github.com/SBOHVM/RPiR/discussions/446

This is the code to make layered plots across multiple blocks from Richards post

**Reply by LollianKosyando** [reply]
Thanks :)

**Comment by LollianKosyando** [comment]
My repository for practical series 02
https://github.com/SBOHVM/lolliankosyandoSeries02.git

**Reply by AnnaKinghorn** [reply]
I can't open it!

**Reply by G-Shiach** [reply]
I also can't, you might need to change the privacy settings to allow us to view it

**Reply by LollianKosyando** [reply]
I think you can access it now:)

**Comment by G-Shiach** [comment]
My new repository for series 03 https://github.com/SBOHVM/GShiachseries03.git

**Comment by AnnaKinghorn** [comment]
My repository for series 3!
https://github.com/SBOHVM/AnnaKinghornSeries3.git

**Comment by Xinyu-GUO-2738378g** [comment]
Here is my repository for series 3 : )
https://github.com/SBOHVM/XinyuSeries03.git
